appveyor.yml

```diff
@@ -3,8 +3,8 @@ environment:
   RUST_BACKTRACE: 1
 
 install:
-  - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-1.22.1-x86_64-pc-windows-gnu.msi"
-  - msiexec /passive /i "rust-1.22.1-x86_64-pc-windows-gnu.msi" ADDLOCAL=Rustc,Cargo,Std INSTALLDIR=C:\Rust
+  - ps: Start-FileDownload "https://static.rust-lang.org/dist/rust-1.23.0-x86_64-pc-windows-gnu.msi"
+  - msiexec /passive /i "rust-1.23.0-x86_64-pc-windows-gnu.msi" ADDLOCAL=Rustc,Cargo,Std INSTALLDIR=C:\Rust
```

Comment:
upstream WR tries to be very careful about not using a released stable rust version until Gecko catches up. We don't need that precaution right now, so we can use the `stable` one. See this for inspiration: https://github.com/gfx-rs/gfx/blob/e21ec9a289fd0e51efd614d7b4004b071cbfedc2/appveyor.yml#L27